plugins/huaweicloud/src/test/resources/huawei_test.json

```diff
@@ -71,7 +71,20 @@
         "storage": [
           "$.storage[0]"
         ],
-        "publicly": true
+        "publicly": true,
```

Comment:
If changing to this userdata way, public ip will be a must :(

Reply:
If something must be installed from the internet, yes. If only configure something, no.